akka-http-core/src/test/scala/akka/http/impl/engine/parsing/RequestParserSpec.scala

```diff
@@ -705,3 +707,57 @@ abstract class RequestParserSpec(mode: String, newLine: String) extends AnyFreeS
 class RequestParserCRLFSpec extends RequestParserSpec("CRLF", "\r\n")
 
 class RequestParserLFSpec extends RequestParserSpec("LF", "\n")
+
+class RequestParserUndefinedSpec extends AnyFreeSpec with Matchers with BeforeAndAfterAll {
```

Comment:
That's the wrong file to test this because it's not about the BodyPartParser. I'll move it to the right place.